samplr/snippet.go

```diff
@@ -536,3 +531,15 @@ func snippetsEquivalent(a SnippetVersion, b SnippetVersion) bool {
 	// Last case, check the file paths are the same
 	return a.File.FilePath == b.File.FilePath && strEq(a.Lines, b.Lines)
 }
+
+func cleanLanguage(language string) string {
+	if language == "C#" {
+		language = "CSHARP"
+	} else if language == "C++" {
+		language = "CPP"
+	}
+	language = strings.ToUpper(language)
+	language = strings.ReplaceAll(language, " ", "_")
```

Comment:
Can we also replace `-` with `_` because that will help these values map to our Language enum values (which use underscores).

Reply:
Done